ngraph/cmake/external_onnx.cmake

```diff
@@ -46,23 +47,39 @@ macro(onnx_set_target_properties)
         target_compile_options(onnx PRIVATE -Wno-unused-variable -Wno-unused-parameter)
         target_compile_options(onnx_proto PRIVATE -Wno-unused-variable)
     endif()
+    set_target_properties(onnx_proto PROPERTIES
+    CXX_VISIBILITY_PRESET default
+    C_VISIBILITY_PRESET default
+    VISIBILITY_INLINES_HIDDEN OFF)
+    target_compile_definitions(onnx PUBLIC ONNX_BUILD_SHARED_LIBS)
+
+    install(TARGETS onnx onnx_proto
+        RUNTIME DESTINATION ${NGRAPH_INSTALL_LIB} COMPONENT ngraph
+        ARCHIVE DESTINATION ${NGRAPH_INSTALL_LIB} COMPONENT ngraph
+        LIBRARY DESTINATION ${NGRAPH_INSTALL_LIB} COMPONENT ngraph)
```

Comment:
do we need to install these libraries to a release package?

Reply:
right, only `onnx_proto` as shared should be part of package